README.md

```diff
@@ -32,9 +40,10 @@ confidentiality for code and data, our initial implementation uses
 Intel® Software Guard Extensions (SGX).
 
 Hyperledger Avalon leverages the existence of a distributed ledger to
- * Maintain a registry of the trusted workers (including their attestation info)
- * Provide a mechanism for submitting work orders from a client(s) to a worker
- * Preserve a log of work order receipts and acknowledgments
+
+- Maintain a registry of the trusted workers (including their attestation info)
+- Provide a mechanism for submitting work orders from a client(s) to a worker
+- Preserve a log of work order receipts and acknowledgments
 
```

Comment:
Are these markdown style changes really needed?  It makes the "blame" searching harder to follow.

Reply:
^^ +1